### PR TITLE
docs(getting-started): Add CDN to getting started documentation

### PR DESCRIPTION
**Pull request by felipesoares6**
Add about CDN at getting started documentation.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -4,16 +4,37 @@ layout: page.jade
...
+<link href="https://unpkg.com/garden@0.34.0/dist/css/garden.min.css" rel="stylesheet">
+```
+
+JavaScript
+
+```html
+  <script scr="https://unpkg.com/garden@0.34.0/dist/js/garden.min.js"></script>
+```
+
+
```
Comment (vitortalaia) [inline docs/getting-started.md]: Uma quebra de linha a mais aqui, não?

**Comment by vitortalaia** [comment]
Acho que vale a pena atualizar o README.md também para adicionar essa opção lá

File: docs/getting-started.md [hunk 2]
```diff
@@ -41,5 +61,4 @@ import file from 'node_modules/garden/src/js/file.js';
 
 JavaScript components rely on jQuery to work, most of the time. Please note `garden.min.js` *does not* comes with jQuery. You have to add your own version of it.
 
-
 ##### Checkout the **<a href="https://github.com/leroy-merlin-br/garden" target="_blank" title="Garden Readme">README</a>** of the project in order how to build the project locally.
```
Comment (willamesoares) [inline docs/getting-started.md]: It seems there is a word missing in this line. Maybe the sentence should be: `Checkout the ... of the project to know how to build the project locally.`

**Comment by willamesoares** [comment]
Will you adjust the files within `dist/js` and `dist/css` in another PR?

File: README.md [hunk 3]
```diff
@@ -29,6 +29,20 @@ npm install garden
...
+CSS
+
+```
+<link href="https://unpkg.com/garden@0.34.0/dist/css/garden.min.css" rel="stylesheet">
+```
+
+JavaScript
+
+```
+<script scr="https://unpkg.com/garden@0.34.0/dist/js/garden.min.js"></script>
```
Comment (fabricionaweb) [inline README.md]: Não precisa especificar a versão. Passa `@latest` ou subtrai que ele pega a ultima

https://unpkg.com/garden/dist/css/garden.min.css
https://unpkg.com/garden/dist/js/garden.min.js

**Comment by felipesoares6** [comment]
@willamesoares  we are having other problems with the minified file, and I'm talking to @leandrogrillo about this.

File: README.md (hunk 3 above)
Comment (felipesoares6) [inline README.md]: I just knew about the @latest, that option without the version sounds much better

File: docs/getting-started.md (hunk 2 above)
Comment (felipesoares6) [inline docs/getting-started.md]: Oh thanks for that, I didn't see it.

**Pull request merged**